frontend/packages/console-shared/src/components/error/error-boundary.tsx

```diff
@@ -15,19 +16,35 @@ export type ErrorBoundaryState = {
 const DefaultFallback: React.FC = () => <div />;
 
 class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoundaryState> {
+  unlisten: () => void = () => {};
+
+  defaultState: ErrorBoundaryState = {
```

Comment:
Does `readonly` work here to make sure we don't accidentally mutate the value?

```suggestion
  readonly defaultState: ErrorBoundaryState = {
```

Reply:
+1 -- TS will definitely get in the way of mutating it accidentally.

Reply:
Added.